code_examples/core_features/linking/04_account_web3name_query_no_sdk.ts

```diff
@@ -1,35 +1,30 @@
-import type { Option, Struct } from '@polkadot/types'
 import type { ApiPromise } from '@polkadot/api'
 import type { KeyringPair } from '@polkadot/keyring/types'
 
-import { hexToString } from '@polkadot/util'
-
 import * as Kilt from '@kiltprotocol/sdk-js'
 
 export async function queryAccountWeb3Name(
   api: ApiPromise,
   lookupAccountAddress: KeyringPair['address']
 ): Promise<Kilt.Did.Web3Names.Web3Name | null> {
   // First RPC call to `api.query.didLookup.connectedDids` for account -> DID lookup.
-  const maybeAccountDid = await api.query.didLookup.connectedDids<
-    Option<Struct>
-  >(lookupAccountAddress)
-  if (maybeAccountDid.isNone) {
+  const accountDid = await api.query.didLookup.connectedDids(
+    lookupAccountAddress
+  )
+  if (accountDid.isNone) {
```

Comment:
Since the SDK has queryWeb3Name(), do we need this whole example? Or does it serve to explain how the internals work?

Reply:
This snippet is an example without the SDK. There is one with and one without.

Reply:
Oh, I didn’t realize that